composer.json

```diff
@@ -26,15 +26,16 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 || ^8.0",
```

Comment:
Please follow the brief as presented in #2 - the additional constraint should be `~8.0.0`.